<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorScript</class>
 <widget class="QMainWindow" name="EditorScript">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор тестов</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Интерпретатор</string>
    </property>
    <addaction name="actionTest"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Окна</string>
    </property>
   </widget>
   <addaction name="menuClose"/>
   <addaction name="menu"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCreate">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Проверка</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Запуск</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
